features/steps/unity_steps.rb

```diff
@@ -1,3 +1,81 @@
+#
+# Mobile steps
+#
+
+When("I wait for the mobile game to start") do
+  # Wait for a fixed time period
+  # TODO: PLAT-6655 Remove the Unity splash screen so we don't have to wait so long
+  sleep 5
+end
+
+When('I relaunch the Unity mobile app') do
+  Maze.driver.launch_app
+  # Wait for a fixed time period
+  sleep 5
+end
+
+When("I run the {string} mobile scenario") do |scenario|
+
+  lookup = {
+      "throw Exception" => 1,
+      "Log error" => 2,
+      "Native exception" => 3,
+      "Log caught exception" => 4,
+      "NDK signal" => 5,
+      "Notify caught exception" => 6,
+      "Notify with callback" => 7,
+      "Change scene" => 8,
+      "Disable Breadcrumbs" => 9,
+      "Start SDK" => 10,
+      "Max Breadcrumbs" => 11,
+  }
+  number = lookup[scenario]
+  $logger.debug "#{scenario}' has dial-in code #{number}"
+
+  step("I dial #{number / 10}")
+  sleep 1
+  step("I dial #{number % 10}")
+  sleep 1
+  step("I press dial")
+  sleep 1
```

Comment:
Are these sleeps necessary for the test to work?

Reply:
I put them in for now just to help gain some confidence in the pressing of the numbered buttons - so that we can see them being entered on the captured video.  It's possibly not needed and certainly one to review over time, although I'm not too concerned given the number of scenarios we currently have in place.